### PR TITLE
[Docs] Add Chinese Js dev doc

### PR DESCRIPTION
**Pull request by cyw3**
- translate 
docs/book/en/src/dev/js/npm.md
to
docs/book/zh/src/dev/js/npm.md

### CODE REVIEW

File: docs/book/zh/src/dev/js/npm.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+
+在 WasmEdge CLI 中运行结果 JS 文件，如下所示。
+
+```bash
+$ wasmedge --dir .:. ../../target/wasm32-wasi/release/wasmedge_quickjs.wasm dist/npm_main.mjs
+md5(message)= 78e731027d8fd50ed642340b7c9a63b3
+sqrt(-4)= 2i
+```
+
+您可以通过这种方式在 WasmEdge 中导入和运行任何纯 JS NPM 包。
```
Comment (alabulei1) [inline docs/book/zh/src/dev/js/npm.md:97]: 您改成你

**Review by alabulei1 — COMMENTED** [review]
@cyw3  Thanks for your contribution! Please check out my comment.

File: docs/book/zh/src/dev/js/npm.md (hunk 1 above)
Comment (cyw3) [reply docs/book/zh/src/dev/js/npm.md]: done

**Comment by alabulei1** [comment]
LGTM. @hydai  could help merge this PR?

**Review by hydai — APPROVED** [review]

**Pull request merged**